src/components/widgets/AutocompleteWidget/AutocompleteWidget.tsx

```diff
@@ -66,29 +66,16 @@ function AutocompleteWidget(props: AutocompleteWidgetProps) {
       return label;
     }
 
-    let hoverText = "";
+    let prefix = value.type === "ontology" 
+      ? value.ontology_name 
+      : value.ontology_name + " > " + value.short_form;
+
+    let hoverText = `Type: ${value.type}\n\nLabel: ${value.label}\n\nPrefix: ${prefix}`;
     if (value.description != undefined) {
-      if (value.type === "ontology") {
-        hoverText = "Type: " + value.type +
-          "\n\nLabel: " + value.label +
-          "\n\nPrefix: " + value.ontology_name +
-          "\n\nDescription: " + value.description;
-      } else {
-        hoverText = "Type: " + value.type +
-          "\n\nLabel: " + value.label +
-          "\n\nPrefix > Short form: " + value.ontology_name + " > " + value.short_form +
-          "\n\nDescription: " + value.description;
-      }
-    } else {
-      if (value.type === "ontology") {
-        hoverText = "Type: " + value.type +
-          "\n\nLabel: " + value.label +
-          "\n\nPrefix: " + value.ontology_name;
-      } else {
-        hoverText = "type: " + value.type +
-          "\n\nLabel: " + value.label +
-          "\n\nPrefix > Short form: " + value.ontology_name + " > " + value.short_form;
```

Comment:
@Pooya-Oladazimi this prefix `> Short form: ` disappeared in the new code, no?

Reply:
@syphax-bouazzouni I removed the text (Prefix -> Short form) only since I thought it would be redundant. The short_form info for the term is there. Up to discussion of course.

Reply:
The text `Prefix > Short form: ` is the label for the informatione, same as `Type: `, `Label: `, and `Description: `.  What follows `Prefix > Short form: ` is first the prefix (e.g. mesh) and then the short form (e.g. C12345). Shouldn't be removed in my opinion. Could be separated if not clear enough.